### DISCUSSION TITLE
remove rarely needed modules from the spigot download

### DISCUSSION THREAD

**Discussion by Wqrld** [EssentialsX/Essentials#4302] [Ideas and Feature Suggestions] [answered]
Almost daily we have people asking why they can't build anymore with essentialsx. While for experienced users it is well-known that this is caused by essentialsantibuild/protect. This is not really clear for "noob" users.

What i would propose is to have the spigot zip only include essentialsX, essentialsXchat, essentialsXspawn (or everything that's not game-breaking if accidentally added: essentialsantibuild and essentialsprotect). And to have the other jars downloadable separately like on the main website.

This would lighten the load for support and also make essentials more easy to use for people that are new to minecraft servers and want some features that essentials has to offer.

**Comment by triagonal** [comment, marked as answer]
This is already planned for the 2.19 release, see this comment and related issue: https://github.com/EssentialsX/Essentials/issues/3889#issuecomment-787472724

**Reply by Wqrld** [reply]
And i thought i looked through all suggestions haha....  Thanks for the answer and the great software :)
